### ISSUE TITLE
Research Catch Tickets

### ISSUE THREAD

**Issue by SaOgaz-NOAA** [nwfsc-fram/boatnet#1058]
- [x] [get permits table sorted out](https://github.com/nwfsc-fram/research-catch-db/issues/10) 
- [x] [Permit Details Screen Tasks](https://github.com/nwfsc-fram/research-catch-db/issues/12)
- [ ] [populate groupings table in dev DB](https://github.com/nwfsc-fram/research-catch-db/issues/18)
- [x] [quasar hint examples](https://github.com/nwfsc-fram/research-catch-db/issues/20)
- [x] Make sure I commit this stuff to the repo... need to strip out DB pw

**Comment by SaOgaz-NOAA** [comment]
Waiting on Kayleigh and Kate to populate the groupings table.  So I'm declaring victory on this ticket for this sprint and will move that subticket to a new issue.
